src/main/java/com/example/bookstore/security/JwtAuthenticationFilter.java

```diff
@@ -18,6 +18,8 @@
 @Component
 @RequiredArgsConstructor
 public class JwtAuthenticationFilter extends OncePerRequestFilter {
+    private static final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
Spring's HttpHeaders class already has this constant, you can also consider using it